backend/api/voting/types.py

```diff
@@ -31,15 +31,27 @@ class VoteType:
     submission: LazyType["Submission", "api.submissions.types"]
 
 
+@strawberry.type
+class RankStat:
+    id: strawberry.ID
+    type: str
+    name: str
+    value: int
+
+
 @strawberry.type
 class RankRequest:
     is_public: bool
     ranked_submissions: List[RankSubmission]
+    stats: List[RankStat]
+
+    @strawberry.field
+    def stat(self, info, type: str) -> Optional[RankStat]:
+        return self.stats.filter(type=type).first()
```

Comment:
I am not sure how this works

Below you create multiple RankStat with the same type, but here you can only return one?

In your code we could have multiple of type `GENDER`